### PR TITLE
Fix Dockerfile issue

### PR DESCRIPTION
**Pull request by BeardedDonut**
The current `Dockerfile` fails when creating the image on multi-arch docker, so I fixed.

### CODE REVIEW

**Comment by Pixep** [comment]
Hey @navid9675 , thanks! I commented your patch

File: Dockerfile [hunk 1]
```diff
@@ -3,7 +3,7 @@ FROM ubuntu:16.04
 MAINTAINER Adrien Leravat <Pixep@users.noreply.github.com>
 
 # Install prerequisites
-RUN echo nameserver 8.8.8.8 > /etc/resolv.conf
+CMD echo nameserver 8.8.8.8 >> /etc/resolv.conf
```
Comment (Pixep) [inline Dockerfile:6]: Are you sure about using `CMD` here? There can only be one CMD, and it is executed when lunched, vs built. I suspect this is actually ignored entirely.

Comment (Pixep) [inline Dockerfile:6]: Could you explain quickly the reasoning for the `>>`? I'm not sure how its going to play with the default `resolv.conf` content.

Comment (BeardedDonut) [reply Dockerfile:6]: Yup, you are correct it will be skipped, I resolved my issue by changing the `FROM ...`.
Thanks

**Pull request closed without merging**